components/DataBrowser/DatasetFiles.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div>
+    <el-breadcrumb separator="/">
+      <el-breadcrumb-item v-for="(item, i) in breadcrumb.slice(0, breadcrumb.length - 1)" :key="i">
+        <span class="hyperlink" @click="handlePath(item.to)">{{ item.label }}</span>
+      </el-breadcrumb-item>
+      <el-breadcrumb-item class="title">{{ title }}</el-breadcrumb-item>
+    </el-breadcrumb>
+
+    <el-table
+      :data="files_data"
+      v-loading="isLoadingFile"
+      element-loading-text="Loading..."
+      element-loading-spinner="el-icon-loading"
+      :cell-style="{padding: '1rem 0 1rem'}"
+    >
+      <el-table-column prop="name" label="Name">
+        <template slot-scope="scope">
+          <span class="table-content">
+            <i class="el-icon-folder" v-if="scope.row.type === 'Folder'"></i>
+            <i class="el-icon-document" v-else></i>
+            <el-popover
+              placement="right"
+              trigger="hover"
+              :content="scope.row.name[0].toUpperCase() + scope.row.name.slice(1)"
+            >
+              <a slot="reference" class="filename-btn" @click="updateFilesData(scope.row.path, scope.row.type)">
+                {{ scope.row.name[0].toUpperCase() + scope.row.name.slice(1) }}
+              </a>
+            </el-popover>
+          </span>
+        </template>
+      </el-table-column>
+      <el-table-column prop="type" label="Type"></el-table-column>
+      <el-table-column label="Action">
+        <template slot-scope="scope">
+          <p v-if="scope.row.type === 'Folder'">-</p>
+          <div v-else>
+            <el-button @click="downloadFile(scope.row.path)" class="icon-btn"><i class="el-icon-download"></i></el-button>
+          </div>
+        </template>
+      </el-table-column>
+    </el-table>
+  </div>
+  
+</template>
+
+<script>
+import backendQuery from '@/services/backendQuery';
+
+export default {
+  name: 'FilesBreadcrumb',
+  data: () => {
+    return {
+      isLoadingFile: false,
+      get_file_path: "",
+      files_data: [],
+      breadcrumb: [
+        {
+          to: 'files',
+          label: 'files'
+        }
+      ],
+    }
+  },
+  
+  created: function() {
+    this.get_file_path = `${this.$config.query_api_url}/collection`;
+    let file_path = `/tempZone/home/rods/12L/datasets/${this.$route.params.id}`;
+    if (this.$route.query.path && this.$route.query.path.length > 6) {
```

Comment:
Can you check rather there is a slash instead of number of characters here instead?